src/validators/validateMinMax.d.ts

```diff
@@ -0,0 +1,6 @@
+declare const validateMinMaxFactory: (
+  min?: number,
+  max?: number,
+  message?: string
+) => (value: any) => string
```

Comment:
hmm so I'm not sure, this factory returns function that takes `any` and returns `string` ? why is the input just `number` ?

Reply:
I think Nick wrote this originally maybe? But `value` could  probably be a raw string or date and then it gets coerced into a string in the implementation. I don't want to change that in this PR though. `String(value` is what it does.

However, I should probably change the return to `string | undefined` to reflect `message : undefined` in  implementation.

Reply:
I think Nick wrote this originally maybe? But `value` could  probably be a raw string or date and then it gets coerced into a string in the implementation. I don't want to change that in this PR though. `String(value` is what it does.

However, I should probably change the return to `string | undefined` to reflect `message : undefined` in  implementation.

Reply:
Update, I've added 
4f98030 and bc6bd08 to make it a bit stronger typed, but I'm certain there are several places this sort of improvement could be made and I'd prefer to have something in and working and then iterate if possible. Does this look good enough?